packages/oidf-resolution-tests/package.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
This had to be a separate package because there is no way I can add the "new JwtService()" agent plugin in package identifier-resolution

Reply:
Then please create a small readme explaining the situation

Reply:
fixed